### PR TITLE
fix: validate email with enter

### LINKED ISSUE #475: Press Enter on "login with magic link" does not do anything
## Steps

1. Go to signin
2. Enter email adress
3. Submit by pressing Enter

Expected behavior: show success message

Currently it does not do anything. In the video we can see the first time where I press enter, and the second time I click on the button.

https://github.com/user-attachments/assets/ca4622aa-70fb-4dd1-bcfc-2947bc28f8fd

### PR DESCRIPTION
**Pull request by spaenleh**
close #475

### CODE REVIEW

File: cypress/e2e/signinWithPassword.cy.ts [hunk 1]
```diff
@@ -41,21 +48,28 @@ describe('Email and Password Validation', () => {
     cy.url().should('contain', redirectionLink);
   });
 
-  it('Sign In With Wrong Password', () => {
+  it.only('Sign In With Wrong Password', () => {
```
Comment (ReidyT) [inline cypress/e2e/signinWithPassword.cy.ts:51]:
```suggestion
  it('Sign In With Wrong Password', () => {
```

Comment (pyphilia) [inline cypress/e2e/signinWithPassword.cy.ts:51]: Remove.

File: src/components/common/Adornments.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+export const EmailAdornment = (
+  <InputAdornment position="start" sx={{ color: 'inherit' }}>
+    <MailIcon color="currentColor" />
+  </InputAdornment>
+);
+export const PasswordAdornment = (
+  <InputAdornment position="start" sx={{ color: 'inherit' }}>
+    <RectangleEllipsisIcon color="currentColor" />
+  </InputAdornment>
+);
```
Comment (pyphilia) [inline src/components/common/Adornments.tsx:14]: Isn't it a better convention to have a single component per file?

**Review by pyphilia — CHANGES_REQUESTED** [review]
LGTM, but you left an `.only`!

**Review by ReidyT — APPROVED** [review]
LGTM 🐦‍🔥

File: cypress/e2e/signinWithPassword.cy.ts (hunk 1 above)
Comment (spaenleh) [reply cypress/e2e/signinWithPassword.cy.ts:51]: 💀

**Review by pyphilia — APPROVED** [review]

**Pull request merged**